src/ui/flutter_app/lib/main.dart

```diff
@@ -115,27 +115,13 @@ Future<void> main() async {
 
 RouteObserver<PageRoute> routeObserver = RouteObserver<PageRoute>();
 
-final globalScaffoldKey = GlobalKey<ScaffoldState>();
-
-class SanmillApp extends StatefulWidget {
-  @override
-  _SanmillAppState createState() => _SanmillAppState();
-}
-
-class _SanmillAppState extends State<SanmillApp> {
-  @override
-  void initState() {
-    super.initState();
-    if (Platform.isWindows) {
```

Comment:
Why was this piece of code removed?

There is a solution to the problem of Audio Player not being supported on Windows, but it is not resolved yet, so a hint to the user is needed.